src/test/ui/const-generics/issues/issue-62220.rs

```diff
@@ -0,0 +1,22 @@
+// check-pass
```

Comment:
```suggestion
// build-pass
```
The ICE happened in codegen which doesn't happen with `--emit metadata` (what `check-pass` implies)

Reply:
Oh, build-pass is a thing too? cool! Changed it :)